src/components/link/Link.vue

```diff
@@ -95,6 +96,13 @@ export default {
             type: String,
             default: null,
         },
+        /**
+         * Override to turn off icons based on stylistic preferences of site.
+         */
+        displayIcon: {
```

Comment:
@chesterroberts-vs Not sure if this new prop is needed as we already have an option of a link with no icon which is just "default" so shouldn't need to pass in another prop?